docs/formatter/black.md

```diff
@@ -866,3 +719,26 @@ with tempfile.TemporaryDirectory() as d1:
     ):
         pass
 ```
+
+### Preserving parentheses around single-element lists
+
+Ruff preserves parentheses around list elements, even if the list only contains a single element. The Black 2025 or newer, on the other hand, removes the parentheses 
+for single-element lists if they aren't multiline and doing so does not change semantics:
+
+```python
+# Input
+items = [(True)]
+items = [(((((True)))))]
+items = {(123)}
+
+# Black
+items = [True]
+items = [True]
+items = {123}
+
+# Ruff
+items = [True]
+items = [True]
+items = {123}
+
```

Comment:
I think this should include the parentheses for the Ruff section:
```py
items = [(True)]
items = [(True)]
items = {(123)}
```

Reply:
Oh yeah. Thanks for catching this